zerver/tests/test_users.py

```diff
@@ -794,6 +830,23 @@ def test_admin_user_can_change_profile_data(self) -> None:
             with self.subTest(field_name=field_dict["name"]):
                 self.assertEqual(field_dict["value"], fields[field_dict["name"]])
 
+        # Check notification
```

Comment:
Also, we should also add check for the notification when removing custom profile fields. I think we test that in `test_custom_profile_fields.py`.